bundle/compliance/cis_gcp/rules/cis_4_1/rule.rego

```diff
@@ -0,0 +1,18 @@
+package compliance.cis_gcp.rules.cis_4_1
+
+import data.compliance.lib.assert
+import data.compliance.lib.common
+import data.compliance.policy.gcp.compute.ensure_default_sa as audit
+import data.compliance.policy.gcp.data_adapter
+
+# Ensure That Instances Are Not Configured To Use the Default Service Account.
+finding = result {
+	# filter
+	data_adapter.is_compute_instance
+
+	# set result
+	result := common.generate_result_without_expected(
+		common.calculate_result(assert.is_false(audit.sa_is_default)),
+		{"Compute instance": input.resource},
```

Comment:
consider returning a more specfic evidence (maybe `resource.data.serviceAccounts`)

Reply:
Not all instances hold a service account, in case they don't the rule will pass.